io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "io.github.NickKarpowicz.LightwaveExplorer",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
```

Comment:
is the GNOME runtime really necessary?

Reply:
The GUI is based on GTK4, which is provided by the gnome runtime